src/components/Todo/common/Todo/Item/index.tsx

```diff
@@ -81,6 +87,91 @@ export const Item: FC<Props> = ({
     setItemText(items[itemIndex].name);
   }, [changeFocus, focus, itemIndex, items]);
 
+  useEffect(() => {
+    setItemText(items[itemIndex].name);
+  }, [items[itemIndex].name]);
+
+  const handleDelete = () => {
+    items.splice(itemIndex, 1);
+    setItemsCallback([...items]);
+  };
+
+  const getNthTodoElement = (n: number) => {
+    return (
+      document.querySelector<HTMLInputElement>(
+        `.${classes.reorderItem}:has(${classes.textFeild}):nth-of-type(${
+          n + 1
+        }) .${classes.textFeild} input`
+      ) ??
+      document.querySelectorAll<HTMLInputElement>(
+        `.${classes.textFeild} input`
+      )[n]
+    );
+  };
+
+  const handleBackspace = () => {
+    if (!itemIndex) return; // Do nothing, for first TODO
+
+    if (inputRef.current!.selectionStart) return; // Do nothing if cursor isn't at start.
+
+    let previousItemElement = getNthTodoElement(itemIndex - 1);
```

Comment:
You can use the predefined setFocus function to focus the element. If you need a selection as well you can add one more method to setSelection. That you can do in different PR if needed.